docs/release-procedure.md

```diff
@@ -0,0 +1,17 @@
+# Release procedure
```

Comment:
A release PR should not happen while there are remaining work items in QA. What if there are items in QA, or other previous merges that we want to omit from the release?

`develop` should be frozen during this time.

Reply:
What do you think about: https://github.com/WildMeOrg/codex-frontend/pull/424/commits/75c6e6145959b784474ec65055f9b13667dd89c1?

Reply:
I was hoping for actual procedures if some commits in `develop` need to be omitted. @brmscheiner, what are your thoughts on this?